presqt/targets/github/tests/views/resource/test_resource_collection.py

```diff
@@ -43,6 +43,23 @@ def test_success_github(self):
         for data in response.data:
             self.assertEqual(len(data['links']), 1)
 
+    def test_success_github_with_search(self):
+        """
+        Return a 200 if the GET method is successful when grabbing GitHub resources with search 
+        parameters.
+        """
+        url = reverse('resource_collection', kwargs={'target_name': 'github'})
+        response = self.client.get(url + '?title=nhl+light', **self.header)
```

Comment:
Let's think a little more about making a highly unique project name to search for.